packages/app/src/systems/Core/components/Header.tsx

```diff
@@ -108,17 +105,22 @@ const styles = {
     paddingLeft: '6px',
   }),
   desktop: cssObj({
+    alignItems: 'center',
     '@xl': {
       display: 'flex',
       alignItems: 'center',
     },
   }),
+  buttonContainer: cssObj({
+    height: '28px',
+  }),
+  iconButton: cssObj({
+    width: '28px',
+  }),
   connectButton: cssObj({
-    ml: '$8',
+    height: 'inherit',
```

Comment:
about this buttons, they should be more similar to mockup:
- border-radius
- size looks different (I see it's same 28px height, but maybe change font/icon size?
- change background to `gray2` and color to `gray11`